pkg/R/RDD.R

```diff
@@ -868,20 +756,15 @@ setMethod("first",
 #'
 #' @param x The RDD to remove duplicates from.
 #' @param numPartitions Number of partitions to create.
-#' @rdname distinct
-#' @export
 #' @examples
 #'\dontrun{
 #' sc <- sparkR.init()
 #' rdd <- parallelize(sc, c(1,2,2,3,3,3))
 #' sort(unlist(collect(distinct(rdd)))) # c(1, 2, 3)
 #'}
-setGeneric("distinct",
-           function(x, numPartitions) { standardGeneric("distinct") })
-
 setClassUnion("missingOrInteger", c("missing", "integer"))
 #' @rdname distinct
-#' @aliases distinct,RDD,missingOrInteger-method
+#' @aliases distinct,RDD-method
 setMethod("distinct",
           signature(x = "RDD", numPartitions = "missingOrInteger"),
```

Comment:
In the master branch the default value here is set to `numPartitions = SparkR::numPartitions(x)`